### DISCUSSION TITLE
后续会支持其他操作系统么？

### DISCUSSION THREAD

**Discussion by yolylight** [flagify-com/OctoMation#100] [General]
CentOS7今年就停止维护了，后续考虑支持Ubuntu或Rocky Linux或者其他发行版么？

**Comment by wuzhi-dev** [comment]
会的 ，计划里ubuntu，麒麟，opencloudos等都会适配的

**Comment by wuzhi-dev** [comment]
> CentOS7今年就停止维护了，后续考虑支持Ubuntu或Rocky Linux或者其他发行版么？

目前centos8和centos stream 8已经支持，可以部署

**Reply by liumzhe299** [reply]
苹果系统能跑起来了吗

**Comment by williamdeve** [comment]
> 苹果系统能跑起来了吗
目前不可以 请使用centos或者opencloudos
